internal/api/user.go

```diff
@@ -144,3 +144,38 @@ func (h *handler) DeleteUser(c *gin.Context) {
 
 	c.JSON(http.StatusOK, response.SuccessResponse(nil))
 }
+
+func (h *handler) ChangePassword(c *gin.Context) {
+	me, err := helper.Me(c)
+	if err != nil {
+		c.JSON(http.StatusNotFound, response.ErrorResponse(response.CodeDefault, response.UserNotFound))
+		return
+	}
+
+	var body struct {
+		OldPassword string `json:"password" binding:"required"`
+		NewPassword string `json:"newpassword" binding:"required" validate:"min=10"`
```

Comment:
```suggestion
		Password string `json:"password" binding:"required"`
		NewPassword string `json:"new_password" binding:"required" validate:"min=10"`
```

Would suggest to name the Parameter like the key in JSON. We also uses underscore for bodies and responses.